src/main/start-main-application/runnables/setup-hostnames.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import electronAppInjectable from "../../electron-app/electron-app.injectable";
+import { beforeElectronIsReadyInjectionToken } from "../runnable-tokens/before-electron-is-ready-injection-token";
+
+const setupHostnamesInjectable = getInjectable({
+  id: "setup-hostnames",
+
+  instantiate: (di) => {
+    const app = di.inject(electronAppInjectable);
+
+    return {
+      id: "setup-hostnames",
+      run: () => {
+        app.commandLine?.appendSwitch("host-rules", [
```

Comment:
Is this `?` here because our override is incomplete?

Reply:
This PR does not override `setupHostnamesInjectable`, what would be a proper way to override it?

Reply:
I would just change the global override for `electronApp` to also include `commandLine: []` or better yet add a new injectable `appendCommandLineSwitch` which then is overridden as a noop.

Reply:
Fixed.